doc/changelog.md

```diff
@@ -375,42 +450,23 @@
   * setting async runtime tokens and values to be errors
   * deallocating live tokens and values
 
-* Fix an issue when no qubit number was specified for the `qinst` primitive. The primitive now
-  correctly deduces the number of qubits when no gate parameters are present. This change is not
-  user facing.
-  [(#496)](https://github.com/PennyLaneAI/catalyst/pull/496)
-
 * Fix the scatter operation lowering when `updatedWindowsDim` is empty.
   [(#475)](https://github.com/PennyLaneAI/catalyst/pull/475)
 
 * Fix the issue in `LightningKokkos::AllocateQubits` with allocating too many qubit IDs on
   qubit re-allocation.
   [(#473)](https://github.com/PennyLaneAI/catalyst/pull/473)
 
-* Add the `wires` property to `catalyst.adjoint` and `catalyst.ctrl`.
+* Fixed an issue where wires was incorrectly set as `<Wires = [<WiresEnum.AnyWires: -1>]>`
+  when using `catalyst.adjoint` and `catalyst.ctrl`, by adding a `wires` property to
+  these operations.
   [(#480)](https://github.com/PennyLaneAI/catalyst/pull/480)
 
-  Without implementing the `wires` property, users would get `<Wires = [<WiresEnum.AnyWires: -1>]>`
-  for the list of wires in these operations.
   Currently, `catalyst.adjoint.wires` supports static wires and workflows with nested branches,
   and `catalyst.ctrl.wires` provides support for workflows with static and variable wires as well as
   nested branches. The `wires` property in `adjoint` and `ctrl` cannot be used in workflows with
   control flow operations.
```

Comment:
This implies that there might be certain usages of `catalyst.adjoint`/`catalyst.ctrl` that are unsupported; do we have any examples we can include?

Reply:
I think this is about the fact that dynamic wires will not be listed in the `.wires` property of the object, rather than limiting the use of `catalyst.adjoint/ctrl` (unless that use involves the wires property).

Reply:
Yeah, I would perhaps lean towards omitting them if they are more confusing rather than providing clarity...